repose-aggregator/components/filters/versioning-filter/src/main/resources/META-INF/schema/config/versioning-configuration.xsd

```diff
@@ -31,49 +31,24 @@
            targetNamespace="http://docs.openrepose.org/repose/versioning/v2.0"
            jaxb:version="2.1">
 
-    <xs:element name="versioning" type="vers-config:ServiceVersionMappingList"/>
+    <xs:element name="versioning" type="ServiceVersionMappingList"/>
 
     <xs:complexType name="ServiceVersionMappingList">
         <xs:annotation>
             <xs:documentation>
-                <html:p>Versioning contains a list of version mappings and service root context.
-                </html:p>
+                <html:p>Versioning contains a list of version mappings and service root context.</html:p>
             </xs:documentation>
         </xs:annotation>
 
         <xs:sequence>
-            <xs:element name="service-root" type="vers-config:ServiceRootContext"/>
-            <xs:element name="version-mapping" type="vers-config:ServiceVersionMapping" minOccurs="1"
-                        maxOccurs="unbounded"/>
+            <xs:element name="version-mapping" type="ServiceVersionMapping" minOccurs="1" maxOccurs="unbounded"/>
         </xs:sequence>
 
         <xs:assert vc:minVersion="1.1"
-                   test="count(distinct-values(vers-config:version-mapping/@id)) = count(vers-config:version-mapping/@id)"
+                   test="count(distinct-values(version-mapping/@id)) = count(version-mapping/@id)"
```

Comment:
Isn't this the thing that @Mario-Lopez found that makes the assertion silently fail (REP-3734)?
I haven't looked yet, but are there any tests around this assertion?